qiita_db/metadata_template/base_metadata_template.py

```diff
@@ -1185,6 +1193,8 @@ def update(self, md_template):
                     "There are no differences between the data stored in the "
                     "DB and the new data provided",
                     QiitaDBWarning)
+                return
```

Comment:
Actually, the one in test_update_fewer_samples is for the warning that missing columns is not raised